python/tox.ini

```diff
@@ -0,0 +1,117 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[tox]
+envlist = py37,py38,py39,linters
+
+[testenv]
+deps =
+    coverage
+    mock
+    nose
+    pytest
+setenv =
+    COVERAGE_FILE = test-reports/{envname}/.coverage
+    PYTEST_ADDOPTS = --junitxml=test-reports/{envname}/junit.xml -vv
+commands =
+    coverage run --source src --parallel-mode -m pytest {posargs}
+    coverage combine
+    coverage report -m
+    coverage html -d test-reports/{envname}/coverage-html
+    coverage xml -o test-reports/{envname}/coverage.xml
+
+[testenv:linters]
+basepython = python3
+skip_install = true
+deps =
+    .
+    {[testenv:flake8]deps}
+    {[testenv:bandit]deps}
+    {[testenv:mypy]deps}
+commands =
+    {[testenv:flake8]commands}
+    {[testenv:bandit]commands}
+    {[testenv:mypy]commands}
+
+[testenv:flake8]
+basepython = python3
+skip_install = true
+deps =
+    flake8>=3.8.4
+    flake8-import-order>=0.9
+    flake8-bugbear
+commands =
+    flake8 src setup.py tests
+
+[testenv:mypy]
+basepython = python3
+skip_install = true
+deps =
+    mypy
+    types-pytz
+    types-python-dateutil
+commands =
+    mypy --ignore-missing-imports src/
+
+[testenv:bandit]
+basepython = python3
+skip_install = true
+deps =
+    bandit
+commands =
+    bandit --ini tox.ini -r src
+
+# [testenv:docs]
+# basepython = python3
+# deps =
+#     -r docs/source/requirements.txt
+# commands =
+#     sphinx-build -E -W -c docs/source/ -b html docs/source/ docs/build/html
+#     sphinx-build -E -W -c docs/source/ -b man docs/source/ docs/build/man
+
+# [testenv:serve-docs]
+# basepython = python3
+# skip_install = true
+# changedir = docs/build/html
+# deps =
+# commands =
+#     python -m http.server {posargs}
```

Comment:
Why is this part of the build commented out?

Reply:
I keep it commented because there are no docs yet and also we need to come up with a plan about python doc.